.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
We'll need molecule tests instead of a travis file + tests/ dir.

Reply:
The .travis.yml and /tests/ dir can be removed if we have molecule tests.

Reply:
Done